rnd/autogpt_server/autogpt_server/app.py

```diff
@@ -1,8 +1,6 @@
 from multiprocessing import set_start_method
 from typing import TYPE_CHECKING
 
-from .util.logging import configure_logging
```

Comment:
In #7981 you moved `configure_logging()` from `main()` to `run_processes()`. Now it's removed entirely. Doesn't this break top-level logging for the application?

Reply:
It's moved to process.py where all the process creation is done.

Reply:
doing it onlu on main() or run_processes() making the test not having it.

Reply:
> It's moved to process.py where all the process creation is done.

Except for the main process, which is what it was in `main()` for.

> doing it only on main() or run_processes() making the test not having it.

It was in every place where we initialize a new process. In `main()`, in `AppProcess.execute_run_command()`, in `Executor.on_node_executor_start` and `Executor.on_graph_executor_start`. It has to be in all these places because the logging config doesn't automatically carry over into newly spawned processes.